src/honeysql_postgres/format.clj

```diff
@@ -42,21 +43,88 @@
 (doseq [[k v] postgres-clause-priorities]
   (register-clause! k v))
 
-(defn get-first [x]
+(defn- get-first
+  "Returns the first element if the passed argument is a collection, else return the passed argument
+   as is."
+  [x]
   (if (coll? x)
     (first x)
     x))
 
+(defn- comma-join-args
+  "Returns the args comma-joined after applying to-sql to them"
+  [args]
+  (if (nil? args)
+    ""
+    (->> args
+         (map to-sql)
+         comma-join
+         paren-wrap)))
+
+;; TODO : Add create-table, drop-table and its clauses to the priority clause map
```

Comment:
@yudistrange I didn't miss it. I was contemplating it might be needed in the map, but have not needed it for the features implemented till now. I would like to keep it that way but just left the todo as a reminder to check for the features yet to be added. Would be removing it before merging.